File: second/java/Plan.java
public class Plan {
    private String name;
    private int cost;
    private String service;

    public Plan(String name, int cost, String service) {
        this.name = name;
        this.cost = cost;
        this.service = service;
    }

    public String getName() {
        return this.name;
    }

    public int getCost() {
        return this.cost;
    }

    public String getService() {
        return this.service;
    }
}

File: second/php/libreria/plan.php
<?php

abstract class Plan{
    private string $name;
    private int $cost;
    private string $service;
    public function __construct(string $name, int $cost, string $service) {
        $this->name = $name;
        $this->cost = $cost;
        $this->service = $service;
    }
    public function getName(): string {
        return $this->name;
    }
    public function getCost(): int {
        return $this->cost;
    }
    public function getService(): string {
        return $this->service;
    }
}

?>